src/flinkSql/flinkLanguageClientManager.ts

```diff
@@ -546,7 +574,7 @@ export class FlinkLanguageClientManager implements Disposable {
             action: "client_initialized",
             compute_pool_id: computePoolId,
           });
-          this.notifyConfigChanged();
+          await this.notifyConfigChanged();
```

Comment:
Make sure we wait for this notification to be sent before simulating the document change. Otherwise, diagnostics might not show up when opening documents every now and then.